src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -276,6 +278,12 @@ namespace winrt::Microsoft::Terminal::Control::implementation
 
         _core = _interactivity.Core();
 
+        // If high contrast mode was changed, update the appearance appropriately.
+        _accessibilitySettings.HighContrastChanged([this](const Windows::UI::ViewManagement::AccessibilitySettings& a11ySettings, auto&&) {

```

Comment:
This is going to explode the moment you close a tab and then the contrast settings change. You're leaving a dangling event handler _with a raw pointer_ to every termcontrol, registered in the AccessibilitySettings.